core/src/main/resources/application.conf

```diff
@@ -1 +1,2 @@
 batchSize = 50000
+akka.http.host-connection-pool.max-retries = 0
```

Comment:
Are we fine with this value? Don't we ever want to retry?

Reply:
That's a really nice question. 
While doing some tests, I noticed that if the API is down and the list of tools is already cached locally, it will retry x times for the tools request before fetching it from the local cache and then it will happen the same for the patterns.
On each retry, the time until next retry doubles. This means that when we already have cache and the API is down, the CLI takes a lot of time to finish.

That's why I decided to remove the retries and if it fails and there is no local cache, the CLI simply fails and you need to re-run the CLI. What's your opinion on this? Do you have any suggestion on how to improve this?

Reply:
I would consider having 1 retry, but I am fine with 0 as long as it is a informed decision that we want to take ;D